src/mapping/PartitionOfUnityMapping.hpp

```diff
@@ -232,7 +232,10 @@ void PartitionOfUnityMapping<RADIAL_BASIS_FUNCTION_T>::computeMapping()
     const auto localNumberOfClusters = clusterIDs.size();
     // Consider the case where we didn't find any cluster (meshes don't match very well)
     if (localNumberOfClusters == 0) {
-      PRECICE_ERROR("Output vertex {} could not be assigned to any cluster. This means that the meshes probably do not match well geometry-wise or the target number of vertices per cluster is too small.", vertex.getCoords());
+      PRECICE_ERROR("Output vertex {} of mesh \"{}\" could not be assigned to any cluster in the rbf-pum mapping. This means that the meshes probably do not match well geometry-wise."
```

Comment:
I guess the default user behavior will be "My meshes are great, this tool does not know what it is saying", before jumping to the only suggested action the error gives. Let's inform the user about the possibility of exporting meshes to check first.

```suggestion
      PRECICE_ERROR("Output vertex {} of mesh \"{}\" could not be assigned to any cluster in the rbf-pum mapping. This probably means that the meshes do not match well geometry-wise: Visualize the exported preCICE meshes to confirm."
```

Reply:
If the meshes are very fine, it might happen that they won't detect mismatches, but then the other options are fine either.